addOns/reports/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Added
+- Add Zap version to html and pdf exports
```

Comment:
`- Add ZAP version to HTML and PDF reports.`

Reply:
Updated the message